### PR TITLE
Remove sample dependency on msal-common

### PR DESCRIPTION
**Pull request by tnorling**
Some of our samples are taking direct dependency on msal-common. This PR refactors these samples to use exports from msal-browser/msal-node instead & adds additional exports to those libraries where needed.

### CODE REVIEW

File: samples/msal-node-samples/ElectronSystemBrowserTestApp/src/CustomLoopbackClient.ts [hunk 1]
```diff
@@ -59,12 +58,12 @@ export class CustomLoopbackClient implements ILoopbackClient {
                     res.end(errorTemplate || "Error occurred loading redirectUrl");
                     reject(new Error('Loopback server callback was invoked without a url. This is unexpected.'));
                     return;
-                } else if (url === CommonConstants.FORWARD_SLASH) {
+                } else if (url === "/") {
```
Comment (peterzenz) [inline samples/msal-node-samples/ElectronSystemBrowserTestApp/src/CustomLoopbackClient.ts:61]: Are you removing the const as well?

**Review by konstantin-msft — APPROVED** [review]

File: change/@azure-msal-browser-1fe30015-5db8-41ae-988f-d27fa55e15e3.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Export IdTokenClaims & PromptValue types",
+  "packageName": "@azure/msal-browser",
+  "email": "thomas.norling@microsoft.com",
+  "dependentChangeType": "patch"
```
Comment (konstantin-msft) [inline change/@azure-msal-browser-1fe30015-5db8-41ae-988f-d27fa55e15e3.json:6]: AFAIK `dependentChangeType` should be `prerelease` as well until we GA.

File: samples/msal-node-samples/ElectronSystemBrowserTestApp/src/CustomLoopbackClient.ts [hunk 3]
```diff
@@ -140,4 +139,63 @@ export class CustomLoopbackClient implements ILoopbackClient {
...
+            throw "Unable to deserialize query string";
+        }
+        return deserializedQueryString;
+    }
+
+    /**
+     * Parses query string from given string. Returns empty string if no query symbol is found.
+     * @param queryString
+     */
+    static parseQueryString(queryString: string): string {
```
Comment (peterzenz) [inline samples/msal-node-samples/ElectronSystemBrowserTestApp/src/CustomLoopbackClient.ts:171]: Regex match of https://domain.com/path?.. or https://domain.com?... etc.  might be safer.

File: change/@azure-msal-browser-1fe30015-5db8-41ae-988f-d27fa55e15e3.json (hunk 2 above)
Comment (tnorling) [reply change/@azure-msal-browser-1fe30015-5db8-41ae-988f-d27fa55e15e3.json:6]: We've never bothered with this field, AFAIK it only impacts where the dependency version bump log appears in the changelog but does not affect the bump itself.

**Review by jo-arroyo — APPROVED** [review]

**Review by peterzenz — APPROVED** [review]

File: samples/msal-node-samples/ElectronSystemBrowserTestApp/src/CustomLoopbackClient.ts (hunk 1 above)
Comment (tnorling) [reply samples/msal-node-samples/ElectronSystemBrowserTestApp/src/CustomLoopbackClient.ts:61]: No, the const is used internally in msal-common/browser/node. I just don't want to demonstrate consuming things directly from common in our samples (and this isn't something we need to or should export from any of our libs)

**Pull request merged**